### PR TITLE
fix(theme-chalk): [rate] fix vertical offset

### LINKED ISSUE #8951: [Component] [rate] el-rate在el-form-item中时样式错误
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.37`
- Element Plus Version: `2.2.9`
- Browser / OS: `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.75 Safari/537.36`
- Build Tool: `Vite`

### Related Component

- `el-rate`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cCBsYW5nPVwidHNcIj5cbmltcG9ydCB7IHJlZiwgdmVyc2lvbiBhcyB2dWVWZXJzaW9uIH0gZnJvbSAndnVlJ1xuaW1wb3J0IHsgdmVyc2lvbiBhcyBFcFZlcnNpb24gfSBmcm9tICdlbGVtZW50LXBsdXMnXG5pbXBvcnQgeyBFbGVtZW50UGx1cyB9IGZyb20gJ0BlbGVtZW50LXBsdXMvaWNvbnMtdnVlJ1xuXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8ZWwtZm9ybT5cbiAgXHQ8ZWwtZm9ybS1pdGVtIGxhYmVsPVwiTGFiZWxcIj5cbiAgICBcdDxlbC1yYXRlIHNob3ctc2NvcmU+PC9lbC1yYXRlPlxuICAgIDwvZWwtZm9ybS1pdGVtPlxuICA8L2VsLWZvcm0+XG5cbiAgPHA+XG4gICAgPGVsLXJhdGUgc2hvdy1zY29yZT48L2VsLXJhdGU+XG4gIDwvcD5cbjwvdGVtcGxhdGU+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

### Steps to reproduce

```html
<el-form>
  <el-form-item label="Label">
    <el-rate show-score></el-rate>
  </el-form-item>
</el-form>

<el-rate show-score></el-rate>
```

### What is Expected?

与el-rate单独使用时样式一致

### What is actually happening?

被el-form-item包裹时样式整体偏上

### Additional comments

_(empty)_

### PR DESCRIPTION
**Pull request by Chuck-Lau**
closed #8951

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by ryuhangyeong — APPROVED** [review]

**Review by xiaoxian521 — COMMENTED** [review]
Maybe it would be better if we gave a default value
`line-height: normal`

File: packages/theme-chalk/src/rate.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@ $rate-height: map.merge(
     font-size: 0;
     vertical-align: middle;
     color: getCssVar('rate-void-color');
-    line-height: initial;
+    line-height: 1;
```
Comment (xiaoxian521) [inline packages/theme-chalk/src/rate.scss:38]: line-height: normal;

**Review by xiaoxian521 — APPROVED** [review]
LGTM

**Review by xiaoxian521 — APPROVED** [review]

**Comment by Tsong-LC** [comment]
> LGTM

Thanks for the review.

**Review by xiaoxian521 — APPROVED** [review]

**Pull request merged**